jekyll/_docs/enterprise/gpu-configuration.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: enterprise
+section: enterprise
+title: "GPU Builder Installation"
+category: [advanced-config]
+order: 6
+description: "How to install CircleCI Enterprise LXC on Ubuntu 14.04"
+---
+
+#GPU Install Instructions 
```

Comment:
This should be an H2, `##`, since the doc title, "GPU Builder Installation" will be rendered as the H1.